pkg/sharing/setup.go

```diff
@@ -114,6 +141,7 @@ func (s *Sharing) AddReplicateTrigger(inst *instance.Instance) error {
 		WorkerType: "share-replicate",
 		Message:    msg,
 		Arguments:  args,
+		Debounce:   "5s",
```

Comment:
What is the purpose of the `debounce` here?

Reply:
The replicator can send several documents at once, so it's more efficient to wait a bit after a change before starting the replicator, because we can group several changes in one replicator execution. For example, a connector can create several bills in the short time period.

Reply:
Ok, smart :)